src/utils/form/control-error.ts

```diff
@@ -1,4 +1,13 @@
+/**
+ * Fields containing the messages to be displayed to the user when an error occurs within a given field.
+ */
 export interface ControlError {
-    message: string;
-    groupMessage?: string;
+    /**
+     * The message specific to and attached to the field
+     */
+    message: string | (() => string);
+    /**
+     * A general error specifying the error and the name of the field affected. Required a ControlLabel in the validator.
```

Comment:
Required a ControlLabel in the validator. << ne devrait pas connaitre ou comprendre un built in validator, j'enleverais cette partie